apps/web/app/(authenticated)/(app)/app/settings/billing/page.tsx

```diff
@@ -250,6 +250,13 @@ const ProUsage: React.FC<{ workspace: Workspace }> = async ({ workspace }) => {
     currentPrice += cost.val.totalCentsEstimate;
     estimatedTotalPrice += forecastUsage(cost.val.totalCentsEstimate);
   }
+  if (workspace.subscriptions?.ratelimits) {
+    const cost = calculateTieredPrices(workspace.subscriptions.ratelimits.tiers, usedRatelimits);
+    if (cost.err) {
+      return <div className="text-red-500">{cost.err.message}</div>;
+    }
+    currentPrice += cost.val.totalCentsEstimate;
```

Comment:
Should `forcastUsage` included here? It is also not included for the `activeKeys` service

Reply:
yes it should, good catch

not for active keys though, as those don't necessarily scale with time